web/gds-user-ui/src/components/Metrics/index.tsx

```diff
@@ -17,7 +17,7 @@ const Metrics = ({ data, type }: MetricsProps) => {
           fallback={
             <Text color={'red'} pt={20}>{t`An error has occurred to load ${type} metric`}</Text>
           }>
-          <SimpleGrid columns={{ base: 4, sm: 2, lg: 4 }} spacingX="30px">
+          <SimpleGrid columns={{ base: 1, sm: 2, lg: 4 }} spacingX="30px">
```

Comment:
This fix was applied after noticing the stats cards appearance wasn't responsive on much smaller screens.